src/external_app/external_messaging.ts

```diff
@@ -104,7 +104,7 @@ export class ExternalMessaging {
     }
     if (window.externalApp) {
       window.externalApp.externalBus(JSON.stringify(msg));
-    } else {
+    } else if (window.webkit) {
```

Comment:
This won't fix it. The problem is I that `externalBus` is not defined because he uses an older iOS app. We should only set up the external bus if we detect that either `externalApp.externalBus` or `webkit!.messageHandlers.externalBus` exists inside `ExternalAuth`.